templates/dashboard/dashboard_sysinfo_about_plugin.html

```diff
@@ -0,0 +1,78 @@
+{if !ConfigHelper::checkPrivilege('hide_sysinfo')}
+<table class="lmsbox" id="lms-ui-dashboard-box-about-lms">
+    <colgroup>
+        <col style="width: 50%;">
+        <col style="width: 50%;">
+    </colgroup>
+    <thead >
+        <tr onclick="showOrHide('dashboard_sysinfo_about_plugin-{$layout.module}');">
```

Comment:
powielasz to co jest na welcomepage?

Reply:
całość bazuje na module welcome - to pozostałość. Można wywalić

Reply:
wywal proszę i zostaw tylko kod ze statami

Reply:
wywalone